include/mptcpd/types.h

```diff
@@ -50,18 +50,39 @@ typedef uint8_t mptcpd_aid_t;
  */
 typedef uint32_t mptcpd_flags_t;
 
-/// Trigger announcement of a new local IP address.
+/**
+ * @brief Trigger announcement of a new local IP address.
+ *
+ * @note Do not use with @c MPTCPD_ADDR_FLAG_FULLMESH.
+ */
 #define MPTCPD_ADDR_FLAG_SIGNAL  (1U << 0)
 
 /// Create a new subflow.
 #define MPTCPD_ADDR_FLAG_SUBFLOW (1U << 1)
 
 /// Set backup priority on the subflow.
 #define MPTCPD_ADDR_FLAG_BACKUP  (1U << 2)
+
+/**
+ * @brief Add remote address to in-kernel fullmesh path management.
+ *
+ * Create a subflow for each remote address through the local address.
```

Comment:
Suggested rephrasing: `If this flag is set, create a subflow connection to each known remote address, originating from this local address. The total number of subflows is subject to the configured limits.`